src/main/kotlin/no/nav/syfo/veileder/VeilederApi.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.syfo.veileder
+
+import io.ktor.server.application.call
+import io.ktor.server.auth.authenticate
+import io.ktor.server.response.respondText
+import io.ktor.server.routing.Routing
+import io.ktor.server.routing.get
+import io.ktor.server.routing.route
+import no.nav.syfo.application.api.auth.JwtIssuerType
+
+const val VEILEDER_LPS_METADATA_PATH = "/api/internad/v3/oppfolgingsplan/lps"
```

Comment:
Vi trenger kanskje ikke begynne med /v3/, da dette blir nytt endepunkt i ny app?

Kan evnt ellers vurdere en lettere path-sti (flisespikkeri :P ), da det blir http://lps-oppfolgingsplan-mottak:team-esyfo/api/internad/v3/oppfolgingsplan/lps

Kanskje det holder med to endepunkter ala
/api/internad/v1/oppfolgingsplan og /api/internad/v1/oppfolgingsplan/{uuid} ?

Reply:
Det var bra du kommenterte, for jeg hadde tenkt til å diskutere det med deg. Jeg var nemlig usikker på hva jeg skulle velge. Jeg tenkte umiddelbart at vi kunne begynne på v1, men så vil det jo for klienten sin del på en måte være en ny versjon av det samme endepunktet, og at det kanskje derfor var naturlig å fortsette med v3.

Modia må fortsette å hente Nav-planene fra en annen app, derfor tenkte jeg at det nok fortsatt burde være lps e.l. med i url'en.

Reply:
Url i gammel app er forøvrig `/api/internad/v2/oppfolgingsplan/lps`

Reply:
Mm. Tenkte at siden man har lps i hostnavnet så var det kanskje ikke nødvendig å ha i path også.

Men det har ikke så mye å si til syvende og sist :P 

Jeg tenker kanskje v1 siden det blir noe nytt, selv om det enn så lenge blir funksjonelt likt.

Reply:
Jeg gidder ikke krangle på det, så da blir det `/api/internad/v1/oppfolgingsplan/lps` og så merger jeg :D